src/PowerShellEditorServices.Protocol/Server/LanguageServer.cs

```diff
@@ -1231,6 +1234,54 @@ public static string GetDecoratedSymbolName(SymbolReference symbolReference)
             await requestContext.SendResult(result);
         }
 
+        protected async Task HandleGetRunspaceRequestAsync(
+            string processId,
+            RequestContext<GetRunspaceResponse[]> requestContext)
+        {
+            var runspaceResponses = new List<GetRunspaceResponse>();
+
+            if (this.editorSession.PowerShellContext.LocalPowerShellVersion.Version.Major >= 5)
+            {
+                if (processId == null) {
+                    processId = "current";
+                }
+
+                var isNotCurrentProcess = processId != null && processId != "current";
```

Comment:
A few parens here would make this a bit easier to visually parse.

Reply:
Also fixed in #881